### ISSUE TITLE
[Bug] winGRASS + Python: 3.9.0 + wxPython: 4.1.1:  location wizard - Vertical alignment flags are ignored with wxEXPAND

### ISSUE THREAD

**Issue by hellik** [OSGeo/grass#1552]
**Describe the bug**

testing winGRASS with Python: 3.9.0, wxPython: 4.1.1, wxWidgets 3.1.5 (possible configuration of OSGeo4W next generation), while using the location wizard to define a new location with an EPSG code, following warning/error message pops up:

```
Starting GRASS GIS...
C:\OSGeo4W\apps\grass\grass78\gui\wxpython\location_wizard\wizard.py:606: wxPyDeprecationWarning: Using deprecated class. Use ItemAttr instead
  self.attr1 = wx.ListItemAttr()
C:\OSGeo4W\apps\grass\grass78\gui\wxpython\location_wizard\wizard.py:608: wxPyDeprecationWarning: Using deprecated class. Use ItemAttr instead
  self.attr2 = wx.ListItemAttr()
Traceback (most recent call last):
  File "C:\OSGeo4W\apps\grass\grass78\gui\wxpython\location_wizard\wizard.py", line 1603, in OnPageChanging
    dlg = SelectTransformDialog(
  File "C:\OSGeo4W\apps\grass\grass78\gui\wxpython\location_wizard\dialogs.py", line 649, in __init__
    bodySizer.Add(
wx._core.wxAssertionError: C++ assertion "!(flags & (wxALIGN_BOTTOM | wxALIGN_CENTRE_VERTICAL))" failed at D:\src\osgeo4w\src\wxwidgets\wxwidgets-3.1.5-0df1d81acd6f1be8624022f8eecb51679008ca40\src\common\sizer.cpp(2184) in wxBoxSizer::DoInsert(): Vertical alignment flags are ignored with wxEXPAND
```

**System description:**

```
GRASS Version: 7.8.5                                                            
Code revision: exported                                                         
Build date: 2021-03-01                                                          
Build platform: x86_64-w64-mingw32                                              
GDAL: 3.2.1                                                                     
PROJ: 8.0.0                                                                     
GEOS: 3.9.1                                                                     
SQLite: 3.33.0                                                                  
Python: 3.9.0                                                                   
wxPython: 4.1.1                                                                 
Platform: Windows-10-10.0.19041-SP0 (OSGeo4W) 

>>> print(wx.version())
4.1.1 msw (phoenix) wxWidgets 3.1.5
```

**Comment by nilason** [comment]
I believe this was addressed with #1330.

**Comment by nilason** [comment]
Closing, please reopen if needed.
